src/utils/db.js

```
@@ -0,0 +1,20 @@
+import 'dotenv/config';
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST || 'localhost',
+  //   user: process.env.DB_USERNAME || 'postgres',
+  //   password: process.env.DB_PASSWORD || '123456',
+  //   database: process.env.DB_DATABASE || 'postgres',
+
+  //   port: process.env.DB_PORT || 5432,
+
+  //   dialect: 'postgres',
+
+  // host: process.env.DB_HOST || 'localhost',
+  username: process.env.DB_USERNAME || 'postgres',
+  password: process.env.DB_PASSWORD || '123456',
+  database: process.env.DB_DATABASE || 'postgres',
```

You provide development fallback defaults for `username`, `password`, and `database`. This is convenient for local dev but be sure to require real credentials in production (e.g., throw or log a clear error when required DB env vars are missing in production). Avoid shipping real credentials in code or repo. Consider failing fast if env vars are absent in production.